task_processing/plugins/mesos/execution_framework.py

```diff
@@ -202,6 +228,17 @@ def get_tasks_to_launch(self, offer):
                 remaining_cpus -= task.cpus
                 remaining_mem -= task.mem
                 remaining_disk -= task.disk
+
+                task_dimensions = {'task_id': task.task_id}
+                get_metric(TASKPROC_TASK_QUEUED_COUNT).count(
+                    -1,
+                    task_dimensions
+                )
+                get_metric(TASKPROC_TASK_QUEUED_TIME_TIMER).record(
+                    (time.time() -
+                     self.task_metadata[task.task_id].time_launched) * 1000,
```

Comment:
I was trying to follow yelp_meteorite's standard that time sent be in milliseconds. I did notice that kew does it seconds however. I suppose this comment is for our own consistency?